<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Meeting Full Details</string>
  </property>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>451</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>C059 [urw]</family>
     <pointsize>28</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Meeting 1 Full Details</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>20</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>C059 [UKWN]</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>1371</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TITLE / LABEL</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string> -    NO TITLE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DATE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>190</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string> -    NO TITLE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TIME STARTED</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string> -    NO TITLE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TIME ENDED</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string> -    NO TITLE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FACILITATOR</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>AGENDA</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
     <pointsize>16</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>PROPOSALS AND DISCUSSIONS</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>571</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier 10 Pitch</family>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>70</y>
     <width>761</width>
     <height>701</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>ATTENDANCE</string>
    </attribute>
    <widget class="QLabel" name="checked_count_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans Mono</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TOTAL ATTENDANCE :     35 OUT OF 50</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>731</width>
       <height>491</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>241</number>
     </attribute>
     <column>
      <property name="text">
       <string>MEMBER</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ATTENDEND</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ON TIME</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>20</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans Mono</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&quot;LATE&quot; PENALTY: KSH. 100</string>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>383</x>
       <y>0</y>
       <width>20</width>
       <height>91</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>60</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans Mono</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&quot;MISSED&quot; PENALTY: KSH. 300</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mark_all_btn">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>131</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Mark All</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>CONTRIBUTIONS</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>731</width>
       <height>521</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>241</number>
     </attribute>
     <column>
      <property name="text">
       <string>MEMBER</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>AMOUNT</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>PAID BY</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="total_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans Mono</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TOTAL CONTRIBUTION:     KSH. XXXX</string>
     </property>
    </widget>
    <widget class="QLabel" name="req_amount">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>10</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans Mono</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>REQUIRED AMOUNT IN KSH</string>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>0</y>
       <width>20</width>
       <height>41</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>6</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>310</y>
     <width>391</width>
     <height>101</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier 10 Pitch</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Change</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
